### PR TITLE
Add update profil method account controller,transfer money in account…

### LINKED ISSUE #10: Create Transfer Method and View
peer to peer Transfer

### PR DESCRIPTION
**Pull request by samkelisojam**
Add  update profile and transfer also their view,

### CODE REVIEW

File: 2024Project-main/2024FinalYearProject/Controllers/AccountController.cs
Comment (Meshack-Mlangeni) [inline 2024Project-main/2024FinalYearProject/Controllers/AccountController.cs:1]: All looks good, if there are any updates/changes, they will be implemented on this.

File: 2024Project-main/2024FinalYearProject/Models/ViewModels/ClientViewModel.cs
Comment (Meshack-Mlangeni) [inline 2024Project-main/2024FinalYearProject/Models/ViewModels/ClientViewModel.cs:1]: Looks good to me

File: 2024Project-main/2024FinalYearProject/Models/ViewModels/UserAccountViewModel.cs
Comment (Meshack-Mlangeni) [inline 2024Project-main/2024FinalYearProject/Models/ViewModels/UserAccountViewModel.cs:1]: @samkelisojam please add property attributes and data annotations , see line 10 to 21: 
eg:
`[Required(ErrorMessage = "Please enter a unique email address")]
        [DisplayName("Email address")]
        [DataType(DataType.EmailAddress)]
        public string Email { get; set; }`

File: 2024Project-main/2024FinalYearProject/Views/Account/UpdateProfile.cshtml
Comment (Meshack-Mlangeni) [inline 2024Project-main/2024FinalYearProject/Views/Account/UpdateProfile.cshtml:1]: looks good to me, @Sabelo90 please confirm.

**Review by Meshack-Mlangeni — CHANGES_REQUESTED** [review]
@samkelisojam please rework

File: 2024Project-main/2024FinalYearProject/Models/ViewModels/UserAccountViewModel.cs
Comment (samkelisojam) [reply 2024Project-main/2024FinalYearProject/Models/ViewModels/UserAccountViewModel.cs:1]: Don't worry about it we deal with that as separate issue soon thank for remind me

**Pull request merged**